day_two_guide/topics/proc_restoring-master.adoc

```diff
@@ -4,8 +4,12 @@ Restoring a master host
 Module included in the following assemblies:
 
 * day_two_guide/host_level_tasks.adoc
+* admin_guide/assembly_restoring-cluster.adoc
 ////
 
+[id='restoring-master_{context}']
+= Restoring a master host backup
+
 After creating a backup of important master host files, if they become corrupted
 or accidentally removed, you can restore the file by copying back the file,
 ensuring it contains the proper content and restart the affected services.
```

Comment:
s/if they become corrupted or accidentally removed, you can restore the file by copying back the file/if it becomes corrupted or accidentally removed, you can restore the file by copying back the file,
s/copying back the file/copying the file back to master ??

Reply:
Line 44; What is an _accidental package_?

Reply:
Line 53: s/Get the differences/Get the differences between the current packages list and the backed-up packages list
Line 57: What is _1a2_?
Line 58: Is that an example output for the command?
Line 64: Is that an example package to restore?

Reply:
33: I don't see one.
44-64: I think it's saying what to do if you can't restart OCP because a package is missing. I'll make some updates.